skyportal/handlers/api/news_feed.py

```diff
@@ -61,13 +61,41 @@ def fetch_newest(model):
                 .all()
             )
 
+        def fetch_newest_comments():
```

Comment:
This query is a duplication of what's implemented above in `fetch_newest`, right? This should be removed, and perhaps just add a check whether `model` is `Comment` in `fetch_newest`, and if so, add the author info. It appears the author info part is the only thing added by this function, but there's a lot of duplication that can be avoided.